### PR TITLE
[CHILI GraFx] Connector Config

### PR DESCRIPTION
**Pull request by BramVerniest**

### CODE REVIEW

File: docs/GraFx-Studio/connectors/connector-acquia-dam/index.md [hunk 1]
```diff
@@ -6,95 +6,101 @@
...
-[See Connector types](/GraFx-Studio/concepts/connectors-media/#support)
+[See Connector types](/GraFx-Studio/concepts/connectors/#types-of-connectors)
 
-## Activation
+## Installation
 
-Once a connector is available in the Connector Hub, it can be activated on your environment.
-
-[Ask CHILI publish](https://mysupport.chili-publish.com/) to activate on your environment.
+[See installtion through Connector Hub](/GraFx-Studio/guides/connector-hub/)
```
Comment (Pieterjan-Blondeel) [inline docs/GraFx-Studio/connectors/connector-acquia-dam/index.md:13]: typo: installation

File: docs/GraFx-Studio/guides/connector-hub/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+![screenshot-full](ch07.png)
+
+Choose the connector you want to deplor or install in your environment.
+
+For detailed configuration parameters, continue on the relevant connector page in documentation.
+
+- [Acquia DAM](/GraFx-Studio/connectors/connect-acquia/)
+
+## Activate / De-activate
```
Comment (Pieterjan-Blondeel) [inline docs/GraFx-Studio/guides/connector-hub/index.md:41]: I would rephrase to how we describe it in the UI: 'available'
And I would add more info about this:

- **Available**: the connector is available in the TD Workspace

**- Not available**:
- ‘not available’ is the [default] state

- if the connector was available in the past but made not available again:
- the connector remains functionally in the WRS & EU Interface ( = we DON’T break existing template)
- but you can’t add a new variable with the connector 
- you can update existing variables with the connector

File: docs/GraFx-Studio/guides/connector-hub/index.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+![screenshot-full](ch03.png)
+
+## Add a connector
+
+Navigate to the list of installed connectors (see above) and click "+ Add connector"
+
+A pop-up window will display.
+
+![screenshot-full](ch04.png)
```
Comment (Pieterjan-Blondeel) [inline docs/GraFx-Studio/guides/connector-hub/index.md:29]: Mss meer uitleg bij Connector Hub screenshot:
- By ... (built by CHILI publish or 3rd party --> https://wonderful-sky-012b9bc03-399.westeurope.1.azurestaticapps.net/GraFx-Studio/concepts/connectors/: Type of Connectors) 
- Media - Data - ... (type connector)

File: docs/GraFx-Studio/guides/connector-hub/index.md
Comment (Pieterjan-Blondeel) [inline docs/GraFx-Studio/guides/connector-hub/index.md:1]: Explain default connector

Per type of connector you have 1 [default] connector and there can only be 1 default connector
[default] connector means that this is the connector that is by default used in the TD Workspace
GraFx Media is the [default] media connector

File: docs/GraFx-Studio/connectors/connector-acquia-dam/index.md (hunk 1 above)
Comment (BramVerniest) [reply docs/GraFx-Studio/connectors/connector-acquia-dam/index.md:13]: corrected

File: docs/GraFx-Studio/guides/connector-hub/index.md (hunk 2 above)
Comment (BramVerniest) [reply docs/GraFx-Studio/guides/connector-hub/index.md:41]: Added the info

File: docs/GraFx-Studio/guides/connector-hub/index.md (hunk 3 above)
Comment (BramVerniest) [reply docs/GraFx-Studio/guides/connector-hub/index.md:29]: done

File: docs/GraFx-Studio/guides/connector-hub/index.md
Comment (BramVerniest) [reply docs/GraFx-Studio/guides/connector-hub/index.md:1]: done
